src/components/AppProvider/AppProvider.scss

```diff
@@ -46,7 +46,7 @@ body {
   min-height: 100%;
   margin: 0;
   padding: 0;
-  background-color: color('sky', 'light');
+  background-color: var(--p-background);
```

Comment:
Does the app provider have access to this variable? It should but it might not.

Reply:
`--p-background` and all the other theme properties get set on the div which is inside the `body` element so css applied to the body won't have access to those custom properties.

Reply:
Lets hard code this value and leave a comment on why it's hard coded. We might come back and fix it.

Reply:
@alex-page updated to hardcode value